flex/views/experiment/ExperimentEditView.mxml

```diff
@@ -1114,17 +1114,6 @@
 					this.userWarningStack.selectedChild = this.userWarningsBoxSampleSetup;
 				} else if (this.isMicroarrayState() && theTab.selectedChild == this.hybSetupView) {
 					this.userWarningStack.selectedChild = this.userWarningsBoxHybSetup
-				} else if (theTab.selectedChild == this.seqSetupView) {
-					if (this.sampleSetupView.currentState == "MDMiSeqState") {
-						seqSetupView.text1.width = 340;
-						seqSetupView.samplePoolingLabel.width = 340;
-					}else if( this.currentState == "NanoStringState") {
-						seqSetupView.isPreppedContainer.visible = false;
-						seqSetupView.isPreppedContainer.includeInLayout = false;
-						seqSetupView.prePooledSamplesHBox.visible = false;
-						seqSetupView.prePooledSamplesHBox.includeInLayout = false;
-					}
-

```

Comment:
You can see from the [diff of John's pull request](https://github.com/hci-gnomex/gnomex/pull/17/files) that this logic was originally implemented via state changes coded in MXML in TabSampleSetupView. When the question was moved to the new tab, the logic was moved to code in ExperimentEditView. When I fixed this bug, I moved the logic to TabSeqSetupView. I made the move because the logic was unique to the tab and it allowed me to update the question number in one place. The logic is still based on the state of the TabSampleSetupView tab, but now that the question is on a new tab, we can't use MXML to manage it.